### ISSUE TITLE
Friendly feedback on accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#42]
Hi there! I love the project and the effort put into it. I wanted to kindly point out that currently, there seems to be a lack of accessibility features. Making the project more accessible could help more people enjoy and benefit from it. Thanks for considering this! Keep up the great work!
